api/src/identity-access-management/infrastructure/repositories/revoked-user-access.repository.js

```diff
@@ -0,0 +1,31 @@
+import { config } from '../../../../src/shared/config.js';
+import { temporaryStorage } from '../../../../src/shared/infrastructure/key-value-storages/index.js';
+import { UserIdIsRequiredError } from '../../domain/errors.js';
+import { RevokeDateMustBeAnInstanceOfDate } from '../../domain/errors.js';
+import { RevokedUserAccess } from '../../domain/models/RevokedUserAccess.js';
+
+const revokedUserAccessTemporaryStorage = temporaryStorage.withPrefix('revoked-user-access:');
+const revokedUserAccessLifespanMs = config.authentication.revokedUserAccessLifespanMs;
+
+const saveForUser = async function (userId, revokeDate) {
```

Comment:
Un nom d'argument `revokeUntil` permettrait de comprendre mieux que cela va définir une période de révocation, sinon le nom `revokeDate` indique plutôt un moment où il se passerait quelque chose.

```suggestion
const saveForUser = async function (userId, revokeUntil) {
```

Reply:
Le nom de l'argument a été modifié :+1: